internal/deviations/deviations.go

```diff
@@ -334,4 +339,6 @@ var (
 	ipv6DiscardedPktsUnsupported = flag.Bool("deviation_ipv6_discarded_pkts_unsupported", false, "Set true for device that does not support interface ipv6 discarded packet statistics, default is false")
 
 	fanOperStatusUnsupported = flag.Bool("deviation_fan_oper_status_unsupported", false, "Device does not support oper-status leaves for some of the fan components. Set this flag to skip checking the leaf.")
+
+	ucmpTrafficTolerance = flag.Float64("deviation_ucmp_traffic_tolerance", 0.2, "Set it to expected ucmp traffic tolerance, default is 0.2")
```

Comment:
NIT: Maybe more specific here, cuz there might be other test cases later that also test about UCMP in some scenarios. how about just say something like `HierarchicalWeightResolutionTolerance`?

Reply:
Fixed it